client/src/Components/UserList.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react'
+import { getUsers } from '../helpers/apiCalls'
+
+
+const UserList = () => {
+  const [ userList, setUserList ] = useState([])
+
+  useEffect(async() => {
+    const resApi = await getUsers()
+    setUserList(resApi)
+  }, [])
+
+  return (
+    <div className='user_list'>
+      {
+        Array.isArray(userList)
```

Comment:
Nice hack to check if the response is an array or object to determine if we received an error.

However: It is a bit more common, that we store received errors in a separate state to decouple it from the data. This way we can have BOTH, the data (e.g. maybe it worked on first request to get the users) + an error (maybe on second request we get an error, because our cookie expired - but we still have the old data to display)

Reply:
I see... I was wondering how to render the specific error you get from the request, and not a general one created by myself . Thanks again!